back/app/services/participants_service.rb

```diff
@@ -1,64 +1,59 @@
 # frozen_string_literal: true
 
 class ParticipantsService
-  ENGAGING_ACTIVITIES = [
```

Comment:
Basically I've removed everything related to the 'activities' table, and rewritten the `participants` method and all methods related to 'counts' to use the new `FactParticipation`. The other methods I've updated to include event attendees and followers, because I don't want to mess up the performance